lib/i18n_routable/config.rb

```diff
@@ -5,6 +5,7 @@ module Config
     self.localize_config = {}
 
     def setup! options
+      I18n.config.enforce_available_locales = false
```

Comment:
I wonder if it might make more sense to just set up the tests to use real locales.